src/Entity/AbstractEntity.php

```diff
@@ -16,6 +16,8 @@
  */
 abstract class AbstractEntity
 {
+    protected $_unknownProperties = array();
```

Comment:
Why the underscore? We can assume that digital ocean is never going to conflict with this anyway.